phoonnx/config.py

```
@@ -150,7 +151,11 @@ def __post_init__(self):
             self.add_diacritics = False
             if self.lang_code and self.lang_code.startswith("ar"):
                 self.add_diacritics = True
-        self.lang_code = self.lang_code or "und"
+
+        try:
+            self.lang_code = standardize_tag(self.lang_code)
+        except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)